code/controllers/subsystem/pai.dm

```diff
@@ -127,6 +142,13 @@ SUBSYSTEM_DEF(pai)
 
 	M << browse(dat, "window=paiRecruit")
 
+/datum/controller/subsystem/pai/proc/get_theme_from_color(color)
+	var/list/themes_by_color = list("#DA1B1B" = "Malware Red", "#2DDF1D" = "Data Green", "#13A5FA" = "Holonet Blue", "#FCEE2E" = "Input Yellow", "#E9901D" = "Storagespace Orange", "#EE1AE3" = "Output Purple")
```

Comment:
Is this the same list you use elsewhere? What happens if someone wants to add a new theme?

Reply:
what do you mean by elsewhere? As far as I am aware its just assigning colors based on picked option?

Reply:
You've got this like, list of theme names the pai can use. It's both as color to theme, theme to color and just theme.
I want adding a new theme color to be as easy as possible. So rather then having the same list but slightly different in 3 different places, I'd like one list that you can modify that's used to create the other two. You'd want to use global lists for this